src/Faithlife.Data/DbConnectorCommand.cs

```diff
@@ -269,6 +274,16 @@ public async ValueTask<DbConnectorResultSets> QueryMultipleAsync(CancellationTok
 		/// </summary>
 		public DbConnectorCommand Cache() => new DbConnectorCommand(Connector, Text, Parameters, isCached: true);
 
+		/// <summary>
+		/// Prepares the command.
+		/// </summary>
+		public DbConnectorCommand Prepare()
+		{
+			using var command = Create();
+			command.Prepare();
```

Comment:
Creating the `IDbCommand`, preparing it, and disposing it does not impact the command created and executed later.

`IDbCommand.Prepare` should be called in `DoCreate`, immediately before it is returned, unless that command was retrieved from the cache.

Reply:
Oops! I forgot that `DbConnector` is just a "connector", not itself an active connection. 🤦🏼